recipe/meta.yaml

```
@@ -1,6 +1,6 @@
 {% set name = "deepspeed" %}
-{% set version = "0.14.0" %}
-{% set number = 1 %}
+{% set version = "0.14.2" %}
+{% set number = 0 %}
```

We'll need to set the build number to 2 here, because there were some CPU builds (with build_number=1) that were accidentally uploaded from the CI at https://github.com/conda-forge/deepspeed-feedstock/pull/59. See https://anaconda.org/conda-forge/deepspeed/files?version=0.14.2

```suggestion
{% set number = 2 %}
```

---

I see, thanks.  I'll update this soon.,